### PR TITLE
(#8131) Bump version Perfetto v21.0

### LINKED ISSUE #8131: [request] perfetto/v21.0
### Package Details
  * Package Name/Version: **perfetto/v21.0**
  * Changelog: **https://github.com/google/perfetto/releases/tag/v21.0**

The above mentioned version is newly released by the upstream project and not yet available as a recipe. PR follows

### PR DESCRIPTION
**Pull request by miklelappo**
Specify library name and version:  **perfetto/21.0**

close #8131

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by toge — DISMISSED** [review]

**Review by madebr — APPROVED** [review]

**Comment by SpaceIm** [comment]
You could take advantage of this PR to improve the CMakeLists which was written for this recipe (cxx_std_11 feature instead of CMAKE_CXX_STANDARD). And also remove pkg_config_name.

**Comment by miklelappo** [comment]
@SpaceIm done!

File: recipes/perfetto/all/CMakeLists.txt [hunk 1]
```diff
@@ -4,12 +4,10 @@ project(perfetto CXX)
 include(conanbuildinfo.cmake)
 conan_basic_setup()
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-
 set(PUBLIC_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder/sdk/perfetto.h)
 
 add_library(perfetto source_subfolder/sdk/perfetto.cc)
+target_compile_features(perfetto PRIVATE cxx_std_11 cxx_constexpr)
```
Comment (SpaceIm) [inline recipes/perfetto/all/CMakeLists.txt:10]: Could you also bump min CMake version to 3.8 please (for cxx_std feature)?

Comment (miklelappo) [reply recipes/perfetto/all/CMakeLists.txt:10]: Done

**Review by SpaceIm — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**